lexical-declarations-es6/README.md

```diff
@@ -0,0 +1,5 @@
+Lexical Declarations (ES6) Sample
+===
+See https://googlechrome.github.io/samples/lexical-declarations-es6/index.html for a live demo.
+
+Learn more at https://www.chromestatus.com/features/4645595339816960
```

Comment:
Can you link to https://www.chromestatus.com/feature/4645595339816960 (`feature`, not `features`) to go directly to the landing page?